docs/source/asr/data/benchmark_en.csv

```diff
@@ -1,7 +1,8 @@
-Model,Dataset,WER (greedy),WER (LM)
-JASPER 10x5 (6 datasets),"Librispeech Dev-Other",10.34,
-JASPER 10x5 (6 datasets),"Librispeech Test-Other",10.50,
-QuartzNet 15x5 (6 datasets),"Librispeech Dev-Other",10.05,
-QuartzNet 15x5 (6 datasets),"Librispeech Test-Other",10.07,
-QuartzNet 15x5-NR (6 datasets),"Librispeech Dev-Other",10.14,
-QuartzNet 15x5-NR (6 datasets),"Librispeech Test-Other",10.11,
+Model Name,Model Base Class,Description
+QuartzNet15x5Base-En,EncDecCTCModel,"QuartzNet15x5 model trained on six datasets: LibriSpeech, Mozilla Common Voice (validated clips from en_1488h_2019-12-10), WSJ, Fisher, Switchboard, and NSC Singapore English. It was trained with Apex/Amp optimization level O1 for 600 epochs. The model achieves a WER of 3.79% on LibriSpeech dev-clean, and a WER of 10.05% on dev-other."
```

Comment:
What is the difference between this and asr_results?
Don't we want to have a table of some results with numbers in the nemo docs? can be some selected metrics?